var/spack/repos/builtin/packages/albany/package.py

```diff
@@ -43,12 +43,21 @@ class Albany(CMakePackage):
             description='Enable MPAS interface in build')
     variant('sfadsize', default='4', values=('4', '6', '8', '12', '24'), multi=False,
             description='SFad size')
+    variant('py',          default=False,
+            description='Enable PyAlbany interface in build')
```

Comment:
After I rebase onto v0.19.0, we're going to have to rebase this and change all `'` to `"` to be consistent with v0.19.0's formatting convention.